Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Ogre Ethnicity

## Changelog:
- Created a custom ethnicity for ogres using `orcish_rework` ethnicity as a template.
- Made a clothing set for ogre cultures using Hindu and Buddhist religious clothes.


# How to test:
There are two ways you can test this:
1. Open up Ruler Designer and generate male and female ogres to see what they look like. The ethnicity is currently called simply `ogre_ethnicity`.
2. Alternatively, load up into a game and look for ogres in-game by using the character search panel.

Give feedback and suggestions on what, if anything, could or should be improved or changed.

## Known issues

- Due to the chonkiness of ogres, some animations cause their hands to clip through their own bodies. This is a problem with Vanilla CK3 and will not be fixed.
